pkg/gadget-service/api/api.pb.go

```diff
@@ -388,6 +388,7 @@ type InfoResponse struct {
 	Version      string `protobuf:"bytes,1,opt,name=version,proto3" json:"version,omitempty"`
 	Catalog      []byte `protobuf:"bytes,2,opt,name=catalog,proto3" json:"catalog,omitempty"`
 	Experimental bool   `protobuf:"varint,3,opt,name=experimental,proto3" json:"experimental,omitempty"`
+	ServerVersion string
```

Comment:
This can't be edited by hand. In order to add a new field, edit the api.proto file around https://github.com/inspektor-gadget/inspektor-gadget/blob/50dd7158e1ba32066471996ff9f117dfb4fcb79b/pkg/gadget-service/api/api.proto#L74-L78
and then run `make generated-files` in the `pkg/gadget-service` folder.